### ISSUE TITLE
Synk says the package's tough-cookie dependency version has a vulnerability in the cookie

### ISSUE THREAD

**Issue by m4bwav** [Swaagie/stackexchange#8]
Here's a post of what synk is saying about tough-cookie:

From https://snyk.io/test/npm/stack-exchange-markdown-retriever

ReDoS via long string of semicolons
high severity

```
Vulnerable module: tough-cookie
Introduced through: stackexchange@0.4.0
```

Detailed paths and remediation

```
Introduced through: stack-exchange-markdown-retriever@1.0.2 › stackexchange@0.4.0 › request@2.72.0 › tough-cookie@2.2.2 Remediation: Run snyk wizard to patch tough-cookie@2.2.2.
```

Overview

tough-cookie package versions 0.9.7 through 2.2.2 are vulnerable to a Regular expression Denial of Service (ReDoS) when long strings of semicolons in the Set-Cookie header, causes the event loop to block for excessive amounts of time.

"The Regular expression Denial of Service (ReDoS) is a Denial of Service attack, that exploits the fact that most Regular Expression implementations may reach extreme situations that cause them to work very slowly (exponentially related to input size). An attacker can then cause a program using a Regular Expression to enter these extreme situations and then hang for a very long time." 1

**Comment by m4bwav** [comment]
Sorry, I was confusing, its request which has the tough-cookie dependency.  You need to bring that one up to, I think, .74

**Comment by Trott** [comment]
Fixed in 1.0.0-1. Use `npm install stackexchange@next` to get it right now or wait until 1.0.0 is published, which should be very soon.
